test/N3Parser-test.js

```diff
@@ -1506,6 +1506,18 @@ describe('Parser', function () {
 
     it('should not parse an invalid ^ path',
       shouldNotParse(parser, '<a>^"invalid" ', 'Expected entity but got literal on line 1.'));
+
+    it('should parse literal as subject',
+        shouldParse(parser, '<a> <evidence> ( <e> [ a <fact>; <gives> {1 <greaterThan> 0}]).',
```

Comment:
I'd suggest a more simple case here to test.

Reply:
Indeed; added a much simpler one 👍